vue-components/src/components/Popover.vue

```diff
@@ -264,6 +440,19 @@ $pointer-edge-length: math.hypot($wikit-Popover-pointer-width/2, $wikit-Popover-
 			}
 		}
 
+		&#{$base}--flush-edges #{$base}__content-wrapper {
+
+			@if $subposition == start {
+				inset-inline-end: min(50% - #{$wikit-Popover-pointer-width} / 2 -
```

Comment:
TODO: Amend the commit message to note that

1. `min()` isn’t supported in IE11, but
2. that’s acceptable (I hope) because that just means it falls back to the non-flush-edges `inset-inline-end`, effectively disabling `avoidOverflowByMakingEdgesFlush()` (so we continue with `avoidOverflowBySettingMaxWidth()`).